### DISCUSSION TITLE
Installation questions

### DISCUSSION THREAD

**Discussion by FletcherJ** [SauceLLC/sauce4zwift#19] [Q&A] [answered]
Greetings:
1) When creating the 2nd zwift account, does the email ID matter?  It implies it should be my name followed by source@internet.com.
2) When creating the account - it walks you through entering the credit card - it might be nice to make it clear that they do not need to do that to create the account.  Zwift doesn't make it easy to realize that.
3) I have the windows version installed.  I can run Zwift.  I can run Sauce.  I can't get them to run together or for Sauce to overlay over zwift (it does a great job overlaying my desktop though).  I can't find any info on what I need to do to get it to overlay on Zwift.

Thanks

**Comment by mayfield** [comment, marked as answer]
Hi @FletcherJ,

1. Email ID doesn't matter and the `source@internet.com` is just a placeholder.  I usually recommend your normal email but add an alias suffix with the + symbol.  For example if I login to Zwift with `justin1234@foobar.com`, I'd use the email `justin1234+sauce@foobar.com` for the Sauce monitor login.  Zwift will treat them as unique accounts.

2. The next release (v0.10.10) has an updated screen which is very emphatic about the 2nd account creation process to help with this.

3. Sounds like Zwift is in fullscreen mode.  In the Zwift settings change it to windowed mode and it should work.   Windows machines can use a tool like https://github.com/Codeusa/Borderless-Gaming to still make the Zwift window appear fullscreen.

**Reply by FletcherJ** [reply]
Ok, I get it - that makes sense. It's just in the YouTube videos, it appears to be full screen, so it didn't occur to me that it wasn't.
